src/components/auth/ResetPassword.js

```diff
@@ -17,15 +17,15 @@ function ResetPassword() {
       const { email } = formData;
 
       if (!isEmail(email)) {
-        setFormState({ error: 'valid email is required' });
-        setTimeout(() => setFormState({}), 2000);
+        setFormState({ error: 'A valid email is required' });
+        setTimeout(() => setFormState({}), 5000);
```

Comment:
What happens if I navigate away from this page before the callback executes? 

IIRC you should get the timeout here (`const timeout = setTimeout(...`), and then clear it in some sort of like dismount callback, `clearTimeout(timeout)`.

Reply:
Fixed!